.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - pwd
   - chmod +x ./scripts/ci/install_myhdl.sh
   - ./scripts/ci/install_myhdl.sh
+  - pip install -e git://github.com/cfelton/rhea.git#egg=rhea
```

Comment:
Why did you add this line back?  I removed it because it is handled in the requirements.txt.

This doesn't match the latest file??  Line 40 doesn't match the base but isn't marked as changed?